app-ios/Sources/CommonComponents/TimetableListItemView.swift

```diff
@@ -38,6 +38,14 @@ public struct TimetableListItemView: View {
                         }
                     }
                 }
+                if item.asSession()?.message != nil {
+                    HStack(spacing: 4) {
+                        Assets.error.swiftUIImage
+                        Text(StringsKt.shared.session_cancel.desc().localized())
+                            .foregroundColor(AssetColors.error.swiftUIColor)
+                            .font(Font.system(size: 12, weight: .regular, design: .default))
+                    }
+                }
```

Comment:
How about combine with above `if let`?
example:

```swift
                if let session = item as? TimetableItem.Session,
                   session.speakers.isEmpty == false {
                    VStack(spacing: 0) {
                        HStack(spacing: 8) {
                            ForEach(session.speakers, id: \.self) { speaker in
                                PersonLabel(name: speaker.name, iconUrl: speaker.iconUrl)
                            }
                        }
                        if let message = session.message {
                            HStack(spacing: 4) {
                                Assets.error.swiftUIImage
                                Text(message...)
                                    .foregroundColor(AssetColors.error.swiftUIColor)
                                    .font(Font.system(size: 12, weight: .regular, design: .default))
                            }
                        }
                    }
                }
```

Reply:
And I think `message` property may be useful.

Reply:
I agree !

[9d7f713](https://github.com/DroidKaigi/conference-app-2022/pull/654/commits/9d7f713fa402a86a468763554e613087b22755cd)